hedera-node/hedera-mono-service/src/main/resources/evm-addresses-blocklist.csv

```diff
@@ -0,0 +1,300 @@
+43cb24e897187c43e77a86b2a4d1195da1540ee7ccf2633f3e109f04293fccf3,Hardhat test account
+0bcb8ba3b59ae835891eafe2e8801d2514153618119b1478d09e2cce5135bb83,Hardhat test account
+5c82a151f730844ce5fc128028e4442f510c6b579d20915fd8cf81e02354b992,Hardhat test account
+216768f6ef69388b3fab11f110b7e1fb08694a95a48f55f6801cdf079e14599c,Hardhat test account
+220e6339b703f4ad85f76e1abfac6f63c444a01baa95631aa00667adba3b7346,Hardhat test account
+3b135f48b6a9219b192ea951ecc79bf77eafa22383f700e498fd295b213c3267,Hardhat test account
+6a9183f26e773765397e86a621efc80df040feeea04e047fb9c8059982b0aad2,Hardhat test account
+5d048e7be67d1528423161ae6420999b4083d11fc9ab1990384de7dc89ddf197,Hardhat test account
+587149c6c919ce9f880eb8a30ab1118aeba31787bda7fee3eb430f9c28488341,Hardhat test account
+addab648d8488d667b01dc9299c3ffe9b0afb3bde5cad43ee6fb705b35578592,Hardhat test account
+18bc325975d0687da38d0f0821573cedc911580aa6e274a731bca216d6069fa7,Hardhat test account
+9165573e1ee13a6c547b793f5632073d9dddb1cf254bbbaf1ea5529172e9554a,Hardhat test account
+4f8a12a5999c1899da1146a141bb6ede9fc2be9d62dbe97e0009e1ccef54f676,Hardhat test account
+12b437ee78c209a7425ba3243280a095795c9ba19e12a67c8fa409bf74f50fe3,Hardhat test account
+ad9919b611791a3126c0541c1d4d63c9d1e45a87c7043df082c212ffe658f03d,Hardhat test account
+bf573e2bc0243ca4c3c4d7dd188a41dd6f612f860f47b07835a5bc3d10d314c0,Hardhat test account
+b357c9a118553574eb1ece22b66abdbfbc482b95cdee6959d371a5f918920e96,Hardhat test account
+c92459e297b8209838f6b0d05eb44cf65d5368b843a216771c712ec41fc46f50,Hardhat test account
+4e56e2ca9b84df3b838c603109065004009b87afa2da911b42ef2c6829181675,Hardhat test account
+6dbfafd104b01ba0d3e7a7552a47eff35e42e2bab98e766a193001c1d1577141,Hardhat test account
+b998b5dc55d07016a3e80bd1ca27418ba2e96f51e15ca94559f3ac4024df725c,Hardhat test account
+7aeca6b611544bdea77d86fcb14c7b8f14633705aacf86bc7486933aec242328,Hardhat test account
+743c4c2a92b09a386ecd033edb71c0bf2b389641ff1aa7fdd7335f70fb2eec30,Hardhat test account
+0b93a86bedcf68f6334b9084036ce739438297b27723a93de2e676c07ac88e74,Hardhat test account
+596b57f2546ba4ca3f20f47c88626673735ed4b90c56d0f6a4637a45498ed52b,Hardhat test account
+f47d65db799c924ef9c4aeb44fc32a7cd9a30ce6b5a67442c89c3935776319da,Hardhat test account
+9a4989c8edad7d649d8f320cc2dae730adeee2bbc4ae37bba86f87bf1b1bc6eb,Hardhat test account
+78c17a46bb4d6b7e620e75443d6db10470d5e3f03334b535fe0967bb88714cc0,Hardhat test account
+fe0c61984318d0b5985f811179fa91b92f53d5d402300735c8eef5c372e2166a,Hardhat test account
+5d21eff406c2fd9487eb327820fddc519f4eef904ba3331911621c52f6e3ffa3,Hardhat test account
+ffef32246e1046f17a86f5f9c68099ffc7a6f55c7625a02084ca3989808a9a84,Hardhat test account
+823d54e5a06d50581e78f4b971093107deb73f17e3d04d5e88efebfcd0ac7042,Hardhat test account
+b592a868a52048890bef3e23dc9b72f8d00d8039ebeec13b18874203f8fdc01a,Hardhat test account
+3a495751c937c4f96a612c765c5900837e641173fdf1e4f32c7e59369c2aba8d,Hardhat test account
```

Comment:
Can add truffle and ganache keys also

Reply:
The 10 accounts `ganache` creates by default (as described [here](https://github.com/trufflesuite/ganache#command-line-use)) are already present in the file (the first 10 `Hedera Local Node address`es).
As for truffle, it seems to be using `ganache` with a different mnemonic, but in the docs [here](https://trufflesuite.com/docs/truffle/how-to/debug-test/use-truffle-develop-and-the-console/) it says the mnemonic is randomly generated when you run it for the first time, so it seems like there are no predefined truffle accounts we can include in the blocklist.